src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -451,6 +467,38 @@ export const handleFormAuthRedirect = [
   _handleFormAuthRedirect,
 ] as ControllerHandler[]
 
+/**
+ * NOTE: This is exported only for testing
+ * Logs user out of SP / CP By deleting cookie
+ * @param authType type of authentication
+ *
+ * @returns 200 with success message when user logs out successfully
+ * @returns 400 if authType is invalid
+ */
+export const _handleFormAuthLogout: ControllerHandler<
+  { authType: AuthType.SP | AuthType.CP },
+  PublicFormAuthLogoutDto
+> = (req, res) => {
+  const { authType } = req.params
+
+  res.clearCookie(JwtName[authType])
+
+  return res.status(200).json({ message: 'Successfully logged out.' })
+}
+
+/**
+ * Handler for /forms/auth/:authType/logout
+ * Valid AuthTypes are SP or CP
+ */
+export const handleFormAuthLogout = [
```

Comment:
nit: strictly speaking this only handles SPCP logout, so this name will become less appropriate as we add more auth types (e.g. SGID). maybe `handleSpcpLogout`?

Reply:
thanks! edited